src/triage/cli.py

```diff
@@ -186,6 +192,14 @@ def experiment(self):
         return experiment
 
     def __call__(self, args):
+
+        if args.setup is not None:
+            logging.info(f"Loading configurations from {args.setup}")
+            spec = importlib.util.spec_from_file_location("triage_config", args.setup)
+            triage_config = importlib.util.module_from_spec(spec)
+            spec.loader.exec_module(triage_config)
+            logging.info(f"Configuration loaded")
```

Comment:
As we discussed, all of this can be moved to somewhere where it will be executed regardless of the subcommand. I think I misunderstood what you were asking, however.

If it's something that should only be done for _certain_ subcommands, then a method on `Triage` class makes sense. Subcommands can invoke it as `self.root.setup()`.

Or, if it's something that should _always_ happen, then perhaps we should instead subclass `Command` to extend `__call__` – or really, the method `call`, which users don't override in the definition of their functionality – *e.g.*:

    class TriageCommand(Command):

        def call(self, *args, **kwargs):
            self._setup(self.args)
            return super().call(*args, **kwargs)

        def _setup(self, args):
            … # what you already have

(That said, I'm not sure that `TriageCommand` is yet needed, for this. And, `argcmdr` should probably make it easier to insert `setup` steps into arbitrary nodes of the command tree.)